src/test/data/JsonAddressBookStorageTest/invalidAndValidPersonAddressBook.json

```diff
@@ -3,11 +3,19 @@
     "name": "Valid Person",
     "phone": "9482424",
```

Comment:
Perhaps the phone value should be changed to match with standard Singapore number format?